### PR TITLE
Authentication screens design

### PR DESCRIPTION
**Pull request by ruchitsimform**
- Add Login screen
- Add Registration screen
- Add Forgot Password screen

### CODE REVIEW

File: app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+class LoginActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityLoginBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        configureFields()
+        setOnClickListenersOnButtons()
```
Comment (yashwantgowla-simform) [inline app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt]: setUpClickListeners is also fine here

File: app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            binding.tlUsername.isErrorEnabled = false
+            binding.tlUsername.error = null
+        }
+        binding.etPassword.addTextChangedListener {
+            binding.tlPassword.isErrorEnabled = false
+            binding.tlPassword.error = null
+        }
+    }
+
+    private fun login() {
```
Comment (yashwantgowla-simform) [inline app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt]: function name should be have some action intent for e.g doLogin(), registerUser() etc

Comment (ruchitsimform) [reply app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt]: Resolved

File: app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt (hunk 1 above)
Comment (ruchitsimform) [reply app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt]: Resolved

**Review by yashwantgowla-simform — APPROVED** [review]

**Pull request merged**